src/components/SearchGridFilter.vue

```diff
@@ -143,9 +145,12 @@ export default {
         { code: 'mccordmuseum', name: 'Montreal Social History Museum' },
         { code: 'met', name: 'Metropolitan Museum of Art' },
         { code: 'museumsvictoria', name: 'Museums Victoria' },
+        { code: 'nhl', name: 'TODO' },
         { code: 'nypl', name: 'New York Public Library' },
         { code: 'rijksmuseum', name: 'Rijksmuseum NL' },
         { code: 'sciencemuseum', name: 'Science Museum – UK' },
+        { code: 'thingiverse', name: 'Thingiverse' },
+        { code: 'WoRMS', name: 'World Register of Marine Species' },
```

Comment:
Is `WoRMS` indeed the `World Register of Marine Species`?

Reply:
That's correct, http://marinespecies.org/